lib/backports/2.6.0/method/compose.rb

```diff
@@ -0,0 +1,11 @@
+unless Method.method_defined? :<<
```

Comment:
Best have independent guards for `:<<` and `:>>`, however unlikely that one would be defined and not the other.